app/helpers/writable_helper.rb

```diff
@@ -44,7 +44,6 @@ def split_paragraphs_largebreak(text)
   # https://apidock.com/rails/ActionView/Helpers/TextHelper/simple_format
   def simple_format_largebreak(text, options={})
     wrapper_tag = options.fetch(:wrapper_tag, :p)
-    text = sanitize(text) if options.fetch(:sanitize, true)
```

Comment:
Can we remove `sanitize: false` from the `sanitize_written_content` thing below accordingly?

Seems to be the only place we use this, so makes sense to me

We can probably remove all the options here as a result actually, and just hard-code wrapper_tag as :p

Reply:
Wait why did I even remove that in the first place

Reply:
I'd guess because this is only called in one place, and always with the same param, but we can add it back to get this merged! =)